src/main/java/com/robothand/highqualitybot/Bot.java

```diff
@@ -15,25 +15,17 @@
  */
 public class Bot {
     static JDA api;
-    public static String PREFIX;
+    public static Config config;
     public static final String VERSION = "0.1";
 
     public static void main(String[] args) {
-        File config;
-        String token;
+        config = null;
 
-        System.out.println("Attempting to read \"config.txt\"");
-        config = new File("config.txt");
-        token = null;
+        System.out.println("Attempting to read \"config.cfg\"");
 
         try {
-            Scanner s = new Scanner(config);
+            config = new Config("config.cfg");
 
-            // token
-            token = s.nextLine();
-
-            // prefix
-            PREFIX = s.nextLine();
 
         } catch (FileNotFoundException e) {
             System.err.println("FATAL: could not find file \"config.txt\"");
```

Comment:
Might be useful to include the current directory in this error message for our friends who don't read documentation